doc/Language/variables.pod6

```diff
@@ -448,6 +459,11 @@ only one (but refers to it twice).  This can be especially significant with cons
 such as C<with>, C<for>, and C<if> that are often used without much consideration of
 the fact that they create blocks.
 
+Just like the C<^>twigil, the C<:> twigil can can be combined with any sigil; using
+C<:> with a sigil will create a formal named parameter of the appropriate type.  Thus
```

Comment:
Again, wouldn't you say "with the proper|corresponding role" would be more precise here than "the appropriate type"?

Reply:
@JJ

The `$` sigil's type constraint isn't a role.

"type constraint" seems to me to be the appropriate generic for this reason alone.

I would also think a newbie is more likely to correctly  guess what a "type constraint" is (a constraint on a variable's type) than guess what a "role" is (a type) *and* guess why it's relevant (it's a type constraint for the associated variable) *and* recognize that `$`'s default type constraint isn't a role.